packages/rancher-pushprox/charts/templates/pushprox-clients.yaml

```diff
@@ -127,7 +127,6 @@ spec:
       - name: metrics-cert-dir-source
         hostPath:
           path: {{ required "Need access to volume on host with the SSL cert files to use HTTPs" .Values.clients.https.certDir }}
-          type: Directory
```

Comment:
do we need to replace this with anything?

Reply:
Nah, removing the type field makes it default to `type: ""`, which is the workaround described in https://github.com/kubernetes/kubernetes/issues/83125